lib/src/bridge/vk_bridge.dart

```diff
@@ -184,6 +184,18 @@ abstract class VKBridge {
   /// Platforms: iOS, Android, Web, Mobile Web
   Future<VKWebAppBoolResult> addToFavorites();
 
+  /// [VKWebAppShowInviteBox] invokes a request window
+  /// for inviting friends to the game.
+  ///
+  /// Platforms: iOS, Android, Web, Mobile Web
+  Future<VKWebAppBoolResult> showInviteBox();
+
+  /// [VKWebAppShowLeaderBoardBox] invokes a request window
+  /// for inviting friends to the game.
+  ///
+  /// Platforms: iOS, Android, Web, Mobile Web
```

Comment:
There is no Web support or there is an error in official documentation.

https://dev.vk.com/bridge/VKWebAppShowLeaderBoardBox